sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/SettingSelector.cs

```diff
@@ -51,7 +57,10 @@ public class SettingSelector
         /// <summary>
         /// Creates a default <see cref="SettingSelector"/> that will retrieve all <see cref="ConfigurationSetting"/> entities in the configuration store.
         /// </summary>
-        public SettingSelector() { }
+        public SettingSelector()
+        {
+            TagsFilter = new ChangeTrackingList<string>();
```

Comment:
Do we need a change tracking list here?   I would think that a simple `List<string>` would be fine.   We're not watching changes over time to build a PATCH request or similar.

Reply:
I've changed this to `List<string>`.